ReactAndroid/src/main/java/com/facebook/react/jscexecutor/OnLoad.cpp

```diff
@@ -19,6 +20,7 @@ namespace {
 
 class JSCExecutorFactory : public JSExecutorFactory {
  public:
+  JSCExecutorFactory(ReactMarker::LogTaggedMarker logTaggedMarker) : logTaggedMarker_(logTaggedMarker) {}
```

Comment:
```suggestion
  JSCExecutorFactory() { logTaggedMarker_ = nullptr; }
  JSCExecutorFactory(ReactMarker::LogTaggedMarker logTaggedMarker) : logTaggedMarker_(logTaggedMarker) {}
```

@axe-fb Maybe something like this? Assuming the other instances of JSC factory that you are talking about don't need marker logging.

Reply:
Yeah. I was thinking we could have a second constructor, which would be hte original signature and hence existing apps will not break.

Reply:
@axe-fb I can make that change but I wonder if that would be enough. This change would not break those features but again it will swallow any marker logs from the features because the logger won't be available. Do you think that would create any problems?